.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+<!--
+  See the contributing guide for detailed guidance about contributing.
+  https://github.com/perses/perses/blob/main/CONTRIBUTING.md
+ -->
+
+# Checklist
```

Comment:
Anything else I'm missing here? I skimmed the contributing guide and didn't see a ton else to add. 

We may want to think about putting together some additional guidance on commit messages and other things as we bring on more contributors, but that work seems separate from this initial addition of a PR template.